integration-tests/gatsby-cli/__tests__/develop.js

```diff
@@ -0,0 +1,76 @@
+import spawn from "cross-spawn"
+import { GatsbyCLI, removeFolder } from "../test-helpers"
+import strip from "strip-ansi"
+
+const timeout = seconds =>
+  new Promise(resolve => {
+    setTimeout(resolve, seconds * 1000)
+  })
+
+const MAX_TIMEOUT = 2147483647
+jest.setTimeout(MAX_TIMEOUT)
+
+describe(`gatsby develop`, () => {
+  const cwd = `gatsby-sites/gatsby-develop`
+
+  beforeAll(() => removeFolder(`${cwd}/.cache`))
+  beforeAll(() => removeFolder(`${cwd}/public`))
+  afterAll(() => removeFolder(`${cwd}/.cache`))
```

Comment:
why do we need this both before and after?

Reply:
do it before just to make sure there wasn't some left over artifacts before. Like if some dev ran gatsby build or something in taht folder before the test. Then also clean up after.

Just trying to make the tests as idempotent as possible.